### PR TITLE
CVPR 2024

### PR DESCRIPTION
**Pull request by libbylbaker**
This pull request adds the CVPR 2024 configuration (see https://cvpr.thecvf.com/Conferences/2024/AuthorGuidelines for the style file).

### CODE REVIEW

File: tueplots/bundles.py [hunk 1]
```diff
@@ -4,6 +4,20 @@
 from tueplots.constants.color import palettes, rgb
 
 
+def cvpr2024(*, column="half", nrows=1, ncols=1, usetex=True, family="serif"):
+    """ICML 2022 bundle."""
```
Comment (pnkraemer) [inline tueplots/bundles.py:8]:
```suggestion
    """CVPR 2024 bundle."""
```

File: tueplots/figsizes.py [hunk 2]
```diff
@@ -253,6 +253,59 @@ def cvpr2022_full(
...
+def cvpr2024_half(
+    *,
+    nrows=1,
+    ncols=1,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """Double-column (half-width) figures for CVPR 2022."""
```
Comment (pnkraemer) [inline tueplots/figsizes.py:265]:
```suggestion
    """Double-column (half-width) figures for CVPR 2024."""
```

File: tueplots/figsizes.py [hunk 3]
```diff
@@ -253,6 +253,59 @@ def cvpr2022_full(
...
+    *,
+    rel_width=1.0,
+    nrows=1,
+    ncols=2,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """Single-column (full-width) figures for CVPR 2022."""
```
Comment (pnkraemer) [inline tueplots/figsizes.py:292]:
```suggestion
    """Single-column (full-width) figures for CVPR 2024."""
```

File: tueplots/fonts.py [hunk 4]
```diff
@@ -51,6 +51,16 @@ def iclr2024(*, family="serif"):
     return _neurips_and_iclr_common(family=family)
 
 
+def cvpr2024_tex(*, family="serif"):
+    """Fonts for ICLR 2024. LaTeX version."""
```
Comment (pnkraemer) [inline tueplots/fonts.py:55]:
```suggestion
    """Fonts for CVPR 2024. LaTeX version."""
```

File: tueplots/fonts.py [hunk 5]
```diff
@@ -51,6 +51,16 @@ def iclr2024(*, family="serif"):
...
+    return _neurips_and_iclr_tex_common(family=family)
+
+
+def cvpr2024(*, family="serif"):
+    """Fonts for ICLR 2023. LaTeX version."""
```
Comment (pnkraemer) [inline tueplots/fonts.py:60]:
```suggestion
    """Fonts for CVPR 2024. LaTeX version."""
```

File: tueplots/figsizes.py [hunk 6]
```diff
@@ -253,6 +253,59 @@ def cvpr2022_full(
...
+    ncols=1,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """Double-column (half-width) figures for CVPR 2022."""
+
+    figsize = _from_base_in(
+        base_width_in=3 + 1 / 4,
```
Comment (pnkraemer) [inline tueplots/figsizes.py:268]:
```suggestion
        base_width_in=3.25,
```

File: tueplots/figsizes.py [hunk 7]
```diff
@@ -253,6 +253,59 @@ def cvpr2022_full(
...
+    ncols=2,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """Single-column (full-width) figures for CVPR 2022."""
+
+    figsize = _from_base_in(
+        base_width_in=6 + 7 / 8,
```
Comment (pnkraemer) [inline tueplots/figsizes.py:295]:
```suggestion
        base_width_in=6.875,
```

**Review by pnkraemer — COMMENTED** [review]
Great, thanks for that contribution!

I think I found a few typos and left a few suggestions. Other than that, it looks good! :)

**Comment by libbylbaker** [comment]
Hi, Yep you're right :) thanks for the suggestions. I've included them now!

**Review by pnkraemer — APPROVED** [review]

**Comment by pnkraemer** [comment]
Thank you!

**Pull request merged**